internal/distro/rhel9/pipelines.go

```diff
@@ -933,6 +933,9 @@ func simplifiedInstallerTreePipeline(repos []rpmmd.RepoConfig, packages []rpmmd.
 		dracutStageOptions.Install = []string{"/fdo_diun_pub_key_root_certs.pem"}
 	}
 	p.AddStage(osbuild.NewDracutStage(dracutStageOptions))
+	if len(greenboot.MonitorServices) != 0 {
+		p.AddStage((osbuild.NewGreenbootConfig(new(osbuild.GreenbootConfig))))
```

Comment:
sshd and networkmanager should be added by default here

Reply:
I think that would be redundant as greenboot is scripted in a way , if it doesn't find the env variable (GREENBOOT_MONITOR_SERVICES) it will default to sshd and networkmanager.

Reply:
uhm, ok even if I now think we need to remove that and have upper layers add services

Reply:
actually, I think we still need to explicitly add them here instead of relying on greenboot default value - we can then drop the default from the package later but at least we know from composer how the system is configured

Reply:
okay that makes sense, added now.